package.json

```diff
@@ -50,7 +54,9 @@
     "asar": "^0.14.3",
     "babel-eslint": "^8.2.5",
     "babel-loader": "^7.1.5",
+    "babel-preset-es2015": "^6.24.1",
     "babel-preset-react-app": "^3.1.2",
+    "babel-preset-stage-3": "^6.24.1",
```

Comment:
Is this still needed?

Reply:
I'm honestly still not 100% sure how the build/live reload process works. You're welcome to try and delete it, or ask @kkennis if he remembers.